pkg/cluster/config_test.go

```diff
@@ -125,9 +125,9 @@ func TestIsValidKubernetesVersion(t *testing.T) {
 		error   bool
 	}{
 		{"cluster version lower than supported version ", "v1.17.1", true},
-		{"cluster version equals supported version", "v1.25.0", false},
-		{"cluster version equals supported version", "v1.26.4", false},
-		{"cluster version greater than supported version ", "v1.27.2", true},
+		{"cluster version equals supported version", "v1.26.0", false},
+		{"cluster version equals supported version", "v1.27.4", false},
+		{"cluster version greater than supported version ", "v1.28.2", true},
```

Comment:
Do we need to keep updating this function given it is just a unit test? Or does the version actually change even with the fake client?  If it does, won't this break the e2e tests on kube version bumps in the SDN repos?

Reply:
No it doesn't change with the fake client, we are not required to do this change. We have already removed the e2e test that tested with the updated version.

Reply:
I suggest leaving it as is then